libafl_bolts/Cargo.toml

```diff
@@ -36,6 +36,7 @@ default = [
   "serdeany_autoreg",
   "alloc",
   "xxh3",
+  "stable_simd",
```

Comment:
Could we only add this to default in non-nightly? That way we would get around the extra dependency

Reply:
Unfortunately, I don't see it is feasible to only enable features for nightly (?). However, do note that `wide` is actually slightly faster than `std::simd`, I think it worths depending on it even on nightly.

Reply:
Ok